GymBuddies/DataModels/Exercise.m

```diff
@@ -0,0 +1,40 @@
+//
+//  Exercise.m
+//  GymBuddies
+//
+//  Created by Gael Rodriguez Gomez on 7/11/22.
+//
+
+#import "Exercise.h"
+
+@implementation Exercise
+
+@dynamic title;
+@dynamic caption;
+@dynamic image;
+@dynamic video;
+@dynamic author;
+@dynamic bodyZoneTag;
+
++ (nonnull NSString *)parseClassName {
```

Comment:
this is good -- can you use this pattern more for any static constant identifiers? or use static constants directly.